src/elements/play-preview.ts

```diff
@@ -66,20 +66,21 @@ export class PlayPreview extends LitElement {
       transition-timing-function: ease-out;
     }
 
+    /* We want the width to be precisely correct since Devvitors get dimension information from context! */
     :host([previewWidth='288']) .preview {
-      width: 288px;
+      width: calc(288px + (var(--border-width) * 2));
```

Comment:
It occurs to me that this may be due to incorrect box-sizing. Maybe change .preview to `box-sizing: content-box` instead (and add a comment if that's the case!).

Reply:
I went back and forth. I normally bias towards border-box everything and this is one of the quirks with that decision. I'm happy to change it though if you prefer!

Reply:
My vote is that content-box would be less confusing than calc but I don't feel strongly about it.

Reply:
Updated!